assets/js/components/DashboardNavigation.js

```diff
@@ -63,6 +64,9 @@ const { useSelect } = Data;
 
 export default function DashboardNavigation() {
 	const dashboardType = useDashboardType();
+	const elementRef = useRef();
+	const { y } = useWindowScroll();
```

Comment:
Thoughts @eugene-manuilov ?

Reply:
@aaemnnosttv @asvinb We already use this same approach in 2 components: 
https://github.com/google/site-kit-wp/blob/bcc13f819025c14da699050fb432eff34b8c0d9a/assets/js/components/EntityHeader.js#L105 https://github.com/google/site-kit-wp/blob/037120d66e456e274b1677527295bd08a3789ea1/assets/js/components/Header.js#L50

Maybe it worth moving this functionality to one of parent components to avoid having it in multiple places? In this case that parent component will add a CSS class (i.e. `googlesitekit-plugin--scrolled`) to the `<div class="googlesitekit-plugin">` element that we can use in CSS to activate certain styles for required elements. 

```scss
.googlesitekit-entity-header {
    .googlesitekit-plugin--scrolled & {
        ...
    }
}
```

Reply:
Thanks @eugene-manuilov . I've simplified the approach for `EntityHeader` based on the latest updates for `DashboardNavigation` and not using the `useWindowScroll` hook within it. Can you review and let me know what you think?